__tests__/examples.test.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path');
+const { promisify: pify } = require('util');
+const { execFile, execFileSync } = require('child_process');
+const execFileAsync = pify(execFile);
+
+describe('examples should work', () => {
+  test('examples/keybinder.js', async done => {
+    const proc = execFile('node', [path.resolve(`${__dirname}/../examples/keybinder.js`)], (err, stdout, stderr) => {
+      expect(err.signal).toBe('SIGTERM');
+      expect(stdout).toBe('<Super>X true\nclicked\nclicked\n');
+      expect(stderr).toBe('');
+      done();
+    });
+
+    await new Promise(resolve => setTimeout(resolve, 1000));
```

Comment:
These 1 second delays can cause problems in large test suites as the delays add up. The test suite runs in ~1.5sec locally for me and this test, by itself, is twice that due to the waiting. I don't think we should keep this in the test suite for reasons explained in my other comment https://github.com/Place1/node-gir/pull/14#discussion_r163795369

but I wanted to comment on these timeouts for future reference :)